modules/node/src/channel/channel.provider.ts

```diff
@@ -71,52 +68,43 @@ class ChannelMessaging extends AbstractMessagingProvider {
 
   async requestCollateral(pubId: string, data: { assetId?: string }): Promise<CFCoreTypes.DepositResult> {
     // do not allow clients to specify an amount to collateralize with
-    return this.channelService.requestCollateral(pubId, getAddress(data.assetId));
+    return (this.channelService.rebalance(
+      pubId,
+      getAddress(data.assetId),
+      RebalanceType.COLLATERALIZE,
+    ) as unknown) as CFCoreTypes.DepositResult;
```

Comment:
should this be `await`ed?

Reply:
You don't technically need a to await if you're returning it since it's still a promise. There's actually a linter rule to explicitly _not_ `return await`: https://eslint.org/docs/rules/no-return-await

Now that I'm looking at that link, it actually makes performance worse to do what we are doing, oops. I'll write this as another issue, yikes.